contributions/2018/Philadelphia-PAWS.Rmd

```diff
@@ -0,0 +1,59 @@
+---
+title: "R-Ladies Philadelphia for Philadelphia Animal Welfare Society (PAWS) | [March-September 2018]"
+author: "R-Ladies Philadelphia"
+output: pdf_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, fig.align="center")
+```
+
+## Groups Involved
+
+### Chapter
+<img src="http://rladiesphilly.org/img/phillylogo.png" width="80" style="float: right;"/> R-Ladies Philadelphia
```

Comment:
Change to:
`<img src="http://rladiesphilly.org/img/phillylogo.png" width="80" style="float: right;"/>`